src/hoi4_world/states/hoi4_states_converter.cpp

```diff
@@ -232,6 +232,12 @@ std::vector<std::set<int>> ConsolidateProvinceSets(std::vector<std::set<int>> co
 }
 
 
+bool IsStateCoastal(const std::set<int>& state_provinces, const std::map<int, std::vector<int>>& coastal_provinces)
+{
+   return false;
```

Comment:
With this function you won't get many docks. :D

Reply:
Who needs ships anyways? :D